src/components/edit-project/editProject.js

```diff
@@ -75,15 +69,17 @@ angular.module("openshiftCommonUI")
                   cb(encodeURIComponent($scope.project.metadata.name));
                 }
 
-                showAlert({
-                  type: "success",
+                NotificationsService.addNotification({
+                  id: 'edit-project-success',
+                  type: 'success',
                   message: "Project \'"  + displayNameFilter(project) + "\' was successfully updated."
                 });
               }, function(result) {
                 $scope.disableInputs = false;
                 $scope.editableFields = editableFields($scope.project);
-                showAlert({
-                  type: "error",
+                NotificationsService.addNotification({
+                  id: 'edit-project-error',
```

Comment:
We probably want to put the project name in the ID. Otherwise if you edit two different projects and get errors for both, you'll only see the first toast notification and not the second.